docker/Dockerfile.php71

```diff
@@ -62,6 +62,9 @@ RUN mkdir --parent /var/log/cron
 ADD configs/cron.conf /etc/nc-cron.conf
 RUN crontab /etc/nc-cron.conf
 
+# Add host IP to /etc/hosts as host.docker.internal. See docker/for-linux#264
+RUN ip -4 route list match 0/0 | awk '{print $3 "host.docker.internal"}' >> /etc/hosts
```

Comment:
That should rather go into bootstrap.sh to make sure it is actually running in the container not during the build of the image.